css/properties/white-space.json

```
@@ -316,7 +316,7 @@
               "chrome": {
                 "version_added": "114",
                 "partial_implementation": true,
-                "notes": "Accepts shorthand values for `white-space-collapse` and `text-wrap` only."
+                "notes": "Accepts shorthand values for `white-space-collapse` and `text-wrap-mode` only."
```

While we're here, could we explicitly add what doesn't work:

```suggestion
                "notes": "Only accepts values for `white-space-collapse` and `text-wrap-mode`, not `white-space-trim`."
```